units/ArmBots/T2/armmav.lua

```diff
@@ -40,6 +40,7 @@ return {
 			normaltex = "unittextures/Arm_normal.dds",
 			subfolder = "armbots/t2",
 			techlevel = 2,
+			rangexpscale = 0.6,
```

Comment:
Is there a better term that can be used than "range XP scale"? It sounds like a value of 1 would be the default, but it looks like a value of 0 would be default?

Reply:
The default seems to be 0.6, as that was what it was in the script before I made changes and it was the variable name as I don't have a large amount of context, or at least didn't when I first started.

Reply:
At the moment I will keep it like that until told otherwise :)

Reply:
I don't mean the value of 0.6 in the armmav unit def, keep that as it is. I mean what would be the implicit value for units that don't have this custom param?

It sounds like this custom param controls getting bonus range with unit XP?